### DISCUSSION TITLE
文档地图加载不出来这个问题什么时候解决？

### DISCUSSION THREAD

**Discussion by Xingen123** [zouyaoji/vue-cesium#312] [General]
![image](https://user-images.githubusercontent.com/29971842/200215236-2399ed24-7a6d-43a8-bbd2-48f1d0ac8305.png)

**Comment by zouyaoji** [comment]
我这没问题哦，你清一下缓存，
如果还不行，文档挂 github page的，说明你那网络github的资源不行。

可以试试 gitee 的，
https://vue-cesium.songluck.com/#/zh-CN/component/controls/vc-overview-map
或者把项目clone到你本地自己运行起来。
